docs/pages/guides/using-custom-fonts.md

```diff
@@ -1,139 +1,102 @@
+To create a new project including this example, run in your terminal:
 
-But, for reference, here is a table of which formats work on which platforms.
+<Terminal cmd={["$ npx create-react-native-app --template with-custom-font"]} />
 
-| Format | Web         | iOS         | Android     |
-| ------ | ----------- | ----------- | ----------- |
-| bdf    | <NoIcon />  | <NoIcon />  | <NoIcon />  |
-| dfont  | <NoIcon />  | <NoIcon />  | <YesIcon /> |
-| eot    | <YesIcon /> | <NoIcon />  | <NoIcon />  |
-| fon    | <NoIcon />  | <NoIcon />  | <NoIcon />  |
-| otf    | <YesIcon /> | <YesIcon /> | <YesIcon /> |
-| ps     | <NoIcon />  | <NoIcon />  | <NoIcon />  |
-| svg    | <YesIcon /> | <NoIcon />  | <NoIcon />  |
-| ttc    | <NoIcon />  | <NoIcon />  | <NoIcon />  |
-| ttf    | <YesIcon /> | <YesIcon /> | <YesIcon /> |
-| woff   | <YesIcon /> | <YesIcon /> | <NoIcon />  |
-| woff2  | <YesIcon /> | <YesIcon /> | <NoIcon />  |
+> **Note**: The above example also uses [`expo-splash-screen`](/versions/latest/sdk/splash-screen/) package. To learn why, read about it more in the [Waiting for fonts to load](#waiting-for-fonts-to-load) section.
 
-### Platform built-in fonts
+## Platform built-in fonts
 
-If you don't want to use a custom font, your best bet is to just use the platform default font by not specifying a font family. Each platform that Expo apps support has a different set of fonts available by default, so there's no good way to specify one that will work everywhere without supplying your own custom font.
+If you don't want to use a custom font, your best bet is to use the platform default font by not specifying a font family. Each platform that Expo apps support has a different set of fonts available by default, so there's no good way to specify one that will work everywhere without supplying your own custom font.
 
 If you are curious, [Nader Dabit](https://twitter.com/dabit3) maintains a [list of fonts always available on iOS and Android](https://github.com/react-native-training/react-native-fonts).
 
-And on web, there are a number of generic font families that you can specify. Different users, browsers, and operating systems will be configured to use different fonts for each of these font family specifications. For example, Safari on an iPhone will use San Francisco as its default for `sans-serif` while Microsoft Edge on Windows will use Arial, and Chrome on Android will typically use Roboto, though OnePlus phones will often use Slate, etc., etc..
+On web, there are a number of generic font families that you can specify. Different browsers, and operating systems are configured to use different fonts for each of these font family specifications. For example, Safari on an iPhone uses San Francisco as its default for `sans-serif` while Microsoft Edge on Windows uses Arial. Similarly, Chrome on Android uses Roboto, though OnePlus phones often use Slate, and so on.
```

Comment:
Perhaps too much info for this doc, but one interesting detail here is that certain/ many (Samsung!) Android phones will let you change the system font, and I think they do that by basically swapping out Roboto. So, even if you say "sans-serif" or "Roboto" specifically, don't be surprised when you get a screenshot of your app in Comic Sans ;-)